lib/game.rb

```diff
@@ -53,61 +53,61 @@ def end_game
     ship_coordinates = nil
     until ship_coordinates != nil do
-      random_coordinates = @computer_data["computer_board"].cells.keys.sample(ship_parameter.length)
-      ship_coordinates = @computer_data["computer_board"].place(ship_parameter, random_coordinates)
+      random_coordinates = @computer_data[:computer_board].cells.keys.sample(ship_parameter.length)
+      ship_coordinates = @computer_data[:computer_board].place(ship_parameter, random_coordinates)
     end
   end
 
   def display_boards
     puts "=============COMPUTER BOARD============="
-    puts @computer_data["computer_board"].render
+    puts @computer_data[:computer_board].render
     puts "=============PLAYER BOARD==============="
-    puts @player_data["player_board"].render(true)
+    puts @player_data[:player_board].render(true)
   end
 
   def player_shot
     puts "Enter the coordinate for your shot:"
-    target = gets.chomp.upcase.delete(" ")[0..1]
+    target_parameter = gets.chomp.upcase.delete(" ")[0..1]
 
-    until @computer_data["computer_board"].valid_target?(target)
+    until @computer_data[:computer_board].valid_target?(target_parameter)
       puts "Please enter a valid coordinate:"
-      target = gets.chomp.upcase.delete(" ")[0..1]
+      target_parameter = gets.chomp.upcase.delete(" ")[0..1]
     end
 
-    @computer_data["computer_board"].cells[target].fire_upon
-    player_feedback(target)
+    @computer_data[:computer_board].cells[target_parameter].fire_upon
+    player_feedback(target_parameter)
   end
 
   def computer_shot
-    target = @player_data["player_board"].cells.keys.sample
+    target_parameter = @player_data[:player_board].cells.keys.sample
 
-    until @player_data["player_board"].valid_target?(target)
-      target = @player_data["player_board"].cells.keys.sample
+    until @player_data[:player_board].valid_target?(target_parameter)
+      target_parameter = @player_data[:player_board].cells.keys.sample
     end
 
-    @player_data["player_board"].cells[target].fire_upon
-    computer_feedback(target)
+    @player_data[:player_board].cells[target_parameter].fire_upon
+    computer_feedback(target_parameter)
   end
 
-  def player_feedback(target)
-    return puts "Your shot on #{target} was a miss." if @computer_data["computer_board"].cells[target].render == "M"
-    return puts "Your shot on #{target} was a hit." if @computer_data["computer_board"].cells[target].render == "H"
-    return puts "Your shot on #{target} sunk my #{@computer_data["computer_board"].cells[target].ship.name}."
+  def player_feedback(target_parameter)
+    return puts "Your shot on #{target_parameter} was a miss." if @computer_data[:computer_board].cells[target_parameter].render == "M"
+    return puts "Your shot on #{target_parameter} was a hit." if @computer_data[:computer_board].cells[target_parameter].render == "H"
+    return puts "Your shot on #{target_parameter} sunk my #{@computer_data[:computer_board].cells[target_parameter].ship.name}."
```

Comment:
If we want to make this more readable, we might consider using a regular ol if-else expression. That way it doesn't run over the 80 character line